packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -62,7 +62,7 @@ export const getCashierValidations = cashier_arr => {
 };
 
 export const excluded_notifications = isMobile()
-    ? ['contract_sold', 'switched_to_real']
+    ? ['contract_sold', 'switched_to_real', 'failed_wallets_upgrade']
```

Comment:
@aizad-deriv In the master branch `const excluded_notifications` looks different and there is `priority_toast_messages`, so when we will merge it in master branch we should be careful, but solution is good in current implementation. JFYI